Formula/tsduck.rb

```diff
@@ -0,0 +1,45 @@
+class Tsduck < Formula
+  desc "MPEG Transport Stream Toolkit"
+  homepage "https://tsduck.io/"
+  url "https://github.com/tsduck/tsduck/archive/v3.28-2551.tar.gz"
+  sha256 "ff39c9b97e17a5ae06556a119e24783427a86b1bb5ac75c7ba0340c7bd07d02d"
+  license "BSD-2-Clause"
+  head "https://github.com/tsduck/tsduck.git", branch: "master"
+
+  depends_on "dos2unix" => :build
+  depends_on "gnu-sed" => :build
+  depends_on "grep" => :build
+  depends_on "openjdk" => :build
+  depends_on "librist"
+  depends_on "pcsc-lite"
+  depends_on "srt"
```

Comment:
```suggestion
  depends_on "srt"
  uses_from_macos "curl"
```
There seem to be errors about a missing `curl-config`.

Reply:
OK, I didn't know "uses_from_macos"